packages/fuchsia_remote_debug_protocol/lib/lib_logging.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
```

Comment:
Would it be alright just to name it logging?  The intent wasn't to prefix it with "lib_" but to say that the file is a library logging utility so that the user can set a logging function for the library/package.

Reply:
Renamed to logging.  I figure the documentation and the fact that it is part of the package should imply that it is for internal logging of the package itself.